packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 47.5.0 - November 7, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763), [Issue #1761](https://github.com/forcedotcom/salesforcedx-vscode/issues/1761))
```

Comment:
Rephrase as:
Update Org Browser to use the current default org and display the correct alias in the org picker